src/frontend/components/UI/Sidebar/components/SidebarLinks/index.tsx

```diff
@@ -220,13 +220,13 @@ export default observer(function SidebarLinks() {
           </Tooltip>
         </div>
         <div className="sidebarLinkGradientWrapper">
-          <Tooltip {...tooltipProps} label="Wiki">
+          <Tooltip {...tooltipProps} label="Docs">
             <NavLink
-              data-testid="wiki"
+              data-testid="Docs"
               className={({ isActive }) =>
                 classNames('Sidebar__item', { active: isActive })
               }
-              to={{ pathname: '/wiki' }}
+              to={{ pathname: '/docs' }}
```

Comment:
we either need to update the route or just keep this as /wiki. Clicking it navigates to a black screen

![image](https://github.com/user-attachments/assets/ce04dc74-80aa-43dc-bf63-e8fe4b12de90)

Reply:
I forgot to change it in the App file, ty!

Reply:
Fixed @BrettCleary